etc/signal-desktop.profile

```diff
@@ -32,4 +34,3 @@ private-dev
 private-tmp
 
 noexec ${HOME}
-noexec /tmp
```

Comment:
Should we keep `noexec /tmp`?

Reply:
Signal doesn't seem to work with noexec /tmp, don't know why